### PR TITLE
Require init on mocha args

### PR DESCRIPTION
**Pull request by 0candy**
#### Related issues
https://github.com/strongloop-internal/scrum-apex/issues/257

### Checklist

- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style
  guide](http://loopback.io/doc/en/contrib/style-guide.html)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "main": "index.js",
   "scripts": {
     "lint": "eslint .",
-    "test": "mocha --timeout 30000",
+    "test": "mocha --timeout 30000 --require test/init.js",
```
Comment (ssh24) [inline package.json:19]: This is interesting. It does not run any tests?

Comment (0candy) [reply package.json:19]: Yes, but yet CI is failing? Same with db2iseries

Comment (ssh24) [reply package.json:19]: Never mind. DB2i does the same.

**Review by ssh24 — APPROVED** [review]
LGTM. Not sure why CI is failing. @0candy Was it failing prior to your change?

**Comment by 0candy** [comment]
@slnode test please

**Comment by ssh24** [comment]
@0candy I do not think adding the change should fail the CI. There is a not necessary line in [`test/init.js`](https://github.com/strongloop/loopback-connector-db2z/blob/master/test/init.js#L8), can you remove this and try with the actual change?

**Comment by 0candy** [comment]
@ssh24 Good catch! That fixed the problem.

**Pull request merged**